data/DataType/boolean_title_case.yaml

```diff
@@ -0,0 +1,6 @@
+name: boolean
+integer-min: '0'
+integer-max: '1'
```

Comment:
Why `integer-max` and `integer-min` for a boolean variable?
You cannot assign 1 to a boolean variable in Pascal.

Reply:
ah ok, I originally didn't.